ad/movement/trusts.md

```diff
@@ -198,6 +208,83 @@ When doing NTLM authentications across trusts, the trusting domain's domain cont
 
 _Nota bene, wether it's Kerberos or NTLM, the ExtraSids are in the same data structure, it's just named differently for each protocol. And, the SID filtering function called by the trusting DC is the same, for both authentication protocols._
 
+## Bastion/Red Forests
+Microsoft introduced Privileged Access Management (PAM) with Server 2016. Among other things, it has very interesting features like -
+    - A bastion forest (Think the administrative forest in ESAE or the famous Red Forest)
```

Comment:
Can you add the expanded acronym in () ? --> `ESAE (Enhanced Security Admin Environment)`

Reply:
Completed!